packages/application-shell/src/components/navbar/menu-items.styles.ts

```diff
@@ -311,6 +321,15 @@ const MenuListItem = styled.li<{
   }
 `;
 
+const SafeArea = styled.span`
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 100%;
+  width: calc(100% - 16px);
```

Comment:
I wonder, where is this `16px` subtraction coming from?

Reply:
This 16 px will always be the padding-left of the navbar area, also the start of the navbar item, whether expanded or collapsed. That is where we want our safe-area to end. 
<img width="252" alt="Screenshot 2024-08-26 at 12 45 15" src="https://github.com/user-attachments/assets/5237bd6a-0f45-43d9-9934-73563f56deea">

Reply:
Thanks got it 👍 Could we also add a comment here explaining that we account for the scrollable menu's padding and use the `spacing30` design token?